features/org.jboss.tools.vpe.feature/feature.xml

```diff
@@ -19,10 +19,6 @@
       %license
    </license>
 
-   <includes
-         id="org.jboss.tools.vpe.preview.feature"
```

Comment:
Richfaces feature currently requires vpe feature. So it will pull in vpe and vpe.preview unless we split these apart.
OR we could exclude richfaces feature from devstudio rpm.